libvast/src/system/index.cpp

```diff
@@ -1242,6 +1262,18 @@ index(index_actor::stateful_pointer<index_state> self,
   self->state.accountant = std::move(accountant);
   self->state.filesystem = std::move(filesystem);
   self->state.catalog = std::move(catalog);
+  self->request(self->state.catalog, caf::infinite, atom::load_v)
+    .await(
```

Comment:
You could (probably should) use the `get_v, taxonomies_v` handler here and reinstate the loading in `spawn_catalog`.